protocol/syncer.go

```diff
@@ -226,10 +251,18 @@ func (s *Syncer) enqueueBlock(peerID peer.ID, b *types.Block) {
 	}
 }
 
+func (s *Syncer) updatePeerStatus(peerID peer.ID, status *Status) {
+	s.logger.Debug("update peer status", "peer", peerID, "number", status.Number, "hash", status.Hash, "difficulty", status.Difficulty)
```

Comment:
Just a bit more info in the logs:
```suggestion
	s.logger.Debug("update peer status", "peer", peerID, "latest block number", status.Number, "latest block hash", status.Hash, "difficulty", status.Difficulty)
```

Reply:
Thank you for good suggestion. Fixed at https://github.com/0xPolygon/polygon-sdk/pull/164/commits/f93af087fb5d9ad4f1e630830d1f989c59657b82